marc.py

```diff
@@ -521,15 +521,7 @@ class MARCExporter(object):
         },
     ]
 
-    SETTINGS = [
-        { "key": STORAGE_PROTOCOL,
-          "label": _("Storage service protocol"),
-          "description": _("Storage protocol to use for uploading generated MARC files. The service must already be configured under 'Storage Services'."),
-          "type": "select",
-          "options": [{ "key": ExternalIntegration.S3, "label": ExternalIntegration.S3 }],
-          "default": ExternalIntegration.S3,
-        },
-    ]
+    SETTING_DESCRIPTION = "Storage protocol to use for uploading generated MARC files. The service must already be configured under 'Storage Services'."
```

Comment:
Should the list of s3 integrations for MARC files be generated here? I'm currently setting it up but in circulation.

Reply:
You could have a method `ExternalIntegration.storage_for(self, _db, artifact_type)`which found all of the storage integrations that were suitable for storing certain things. "Suitability" would be hard-coded -- we know that an S3 integration can be used for anything so long as you've defined an appropriate bucket for that purpose, and we know that an LCP integration can only be used to store ebooks. I think this code could definitely go into core.

Reply:
That makes sense but I'm giving the type of bucket each integration is used for when it is added to a collection or catalog service (it's "purpose"). So here, it wouldn't be able to know that some buckets are specifically for, say, MARC files. In circulation, I was listing all the available buckets regardless of how they could be used.

Reply:
That's fine -- I'm only suggesting you be able to grab the buckets that _could_ be used for MARC files because it's technically possible.